iphone/Classes/TiAppiOSUserNotificationCenterProxy.m

```diff
@@ -145,19 +152,20 @@ - (void)removeDeliveredNotifications:(id)args
         return;
       }
 
-      [center getPendingNotificationRequestsWithCompletionHandler:^(NSArray<UNNotificationRequest *> *requests) {
-        // Loop through current notification requests
+      [center getDeliveredNotificationRequestsWithCompletionHandler:^(NSArray<UNNotificationRequest *> *requests) {
```

Comment:
@janvennemann  In callback this method returns " NSArray<UNNotification *> * _Nonnull notifications" rather  "NSArray<UNNotificationRequest *> *requests" . See doc [here](https://developer.apple.com/documentation/usernotifications/unusernotificationcenter/1649520-getdeliverednotificationswithcom?language=objc). Please update the implementation accordingly. Thanks!

Reply:
Oh, the method name was also wrong, fixed the issues!